kotlin-android/app/src/main/java/com/mongodb/tasktracker/LoginActivity.kt

```diff
@@ -68,7 +68,7 @@ class LoginActivity : AppCompatActivity() {
 
         if (createUser) {
             // register a user using the Realm App we created in the TaskTracker class
-            taskApp.emailPasswordAuth.registerUserAsync(username, password) {
+            taskApp.emailPassword.registerUserAsync(username, password) {
```

Comment:
Question: where does the `taskApp` get declared?

Reply:
You can find it in this tasktracker's Application [subclass](https://github.com/nathan-contino-mongo/realm-tutorial/blob/phase2-android-beta8/kotlin-android/app/src/main/java/com/mongodb/tasktracker/TaskTracker.kt#L12)
(which is actually a best practice recommended in our [quickstart](https://docs.mongodb.com/realm/android/quick-start/#initialize-realm), since Apps are expensive and should generally be handled as singletons, and you should only `init` realm a single time per run)

Reply:
I see, I was reading about global vars for kotlin to see if I was missing something and came across Kotlin companion objects which seems like an idiomatic pattern. Have you considered using that pattern?